src/sentry/issues/priority.py

```diff
@@ -58,6 +62,11 @@ def update_priority(
             "reason": reason,
         },
     )
+    # if the group corresponds to a metric issue, then update its incident activity
+    # we will remove this once we've fully deprecated the Incident model
```

Comment:
might be nice to have a specific `TODO - <easy_to_search_for_string>` here as well, so we can quickly update the cases as we drop the incident model.